.tekton/kogito-serverless-operator-pull-request.yaml

```diff
@@ -0,0 +1,488 @@
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.openshift.io/repo: https://github.com/kiegroup/kogito-serverless-operator?rev={{revision}}
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-cel-expression: event == "pull_request" && target_branch
+      == "main"
+  creationTimestamp: null
+  labels:
+    appstudio.openshift.io/application: operator
+    appstudio.openshift.io/component: kogito-serverless-operator
+    pipelines.appstudio.openshift.io/type: build
+  name: kogito-serverless-operator-on-pull-request
+  namespace: dhanak-tenant
+spec:
+  params:
+  - name: git-url
+    value: '{{source_url}}'
+  - name: revision
+    value: '{{revision}}'
+  - name: output-image
+    value: quay.io/redhat-user-workloads/dhanak-tenant/kogito-serverless-operator:on-pr-{{revision}}
+  - name: image-expires-after
+    value: 5d
+  - name: dockerfile
+    value: Dockerfile
```

Comment:
@mareknovotny @rgdoliveira since ceit is not yet supported, here we would define a Dockerfile path for the operator image.